src/nupic/algorithms/Connections.cpp

```diff
@@ -39,8 +39,23 @@ using namespace nupic::algorithms::connections;
 
 Connections::Connections(CellIdx numCells) { initialize(numCells); }
```

Comment:
question aside of this PR: how does the numCells in Connections translate to numColumns in SP/TP? Is there the concept of numCols and numCellsPerCol?

Reply:
A mini-column in the SP  **is** a cell in the Connections.  Two words for the same data structure.  

Now conceptually a mini-column has multiple cells, but that is dealt with by the TemporalMemory when it models the cells in each mini-column.

Reply:
maybe a note of this could be added to SP.hpp, where there was/is a concept of Columns, which in Conn are called Cells (while TP/TM uses (micro)columns + cellsPerCol)